pkgs/applications/misc/passwordsafe/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, meson, ninja, pkg-config, gettext, fetchFromGitLab
+, python3, libhandy, libpwquality
+, wrapGAppsHook, gtk3, glib, gobject-introspection
+, desktop-file-utils, appstream-glib, gnome3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "passwordsafe";
+  version = "3.99.2";
+
+  format = "other";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "World";
+    repo = "PasswordSafe";
+    rev = version;
+    sha256 = "0pi2l4gwf8paxm858mxrcsk5nr0c0zw5ycax40mghndb6b1qmmhf";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    gettext
+    pkg-config
+    wrapGAppsHook
+    desktop-file-utils
+    appstream-glib
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    gnome3.adwaita-icon-theme
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pygobject3
+    libkeepass
```

Comment:
I don't see `import libkeepass` anywhere in the source https://gitlab.gnome.org/search?utf8=%E2%9C%93&search=libkeepass&group_id=362&project_id=4356&scope=&search_code=true&snippets=false&repository_ref=3.99.2&nav_source=navbar

From looking at `meson.build` it only uses pykeepass.

Reply:
Yes, it looks like pykeepass is the successor and libkeepass is deprecated, thanks